src/test/kotlin/uk/gov/justice/digital/hmpps/mailboxregisterapi/integration/mailboxes/omumailboxes/ListingOmuMailboxesTest.kt

```diff
@@ -58,8 +58,67 @@ class ListingOmuMailboxesTest : IntegrationTestBase() {
       .expectBody(object : ParameterizedTypeReference<List<OffenderManagementUnitMailbox>>() {})
       .returnResult().responseBody!!
 
-    Assertions.assertThat(mailboxes).hasSize(2)
+    Assertions.assertThat(mailboxes).hasSize(5)
```

Comment:
Not a blocker, but it's a bit of a convention to [static import](https://stackoverflow.com/a/31303325) these `assertThat` methods